ArduCopter/mode_poshold.cpp

```diff
@@ -156,25 +156,13 @@ void Copter::ModePosHold::run()
         target_climb_rate = get_pilot_desired_climb_rate(channel_throttle->get_control_in());
         target_climb_rate = constrain_float(target_climb_rate, -get_pilot_speed_dn(), g.pilot_speed_up);
 
+        // initiate take-off
+        if (!takeoff.running()) {
```

Comment:
This is missing the call to `triggered`, it should be:
```suggestion
        if (takeoff.triggered() && !takeoff.running()) {
```

Reply:
Fixed.